bellboy/actors/lead.py

```diff
@@ -101,6 +120,18 @@ def receiveMsg_SensorEventMsg(self, message, sender):
             self.log.debug("received 3 events, turning off sensor.")
             self.send(self.ultrasonic_sensor, SensorReq.STOP)
 
+    def receiveMsg_LCDResp(self, sender):
+        self.log.info(
+            str.format("Received message from" , self.nameOf(sender))
+        )
+
+    def receiveMsg_LCDEventMsg(self, message, sender):
```

Comment:
LCD prob wont generate events cus its an output device, this function isnt needed